src/System.Drawing.Common/tests/Printing/MarginsTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿//
+// System.Drawing.Printing.Margins unit tests
+//
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Make sure this header is at the very top. You can just delete the lines above in this file.